### PR TITLE
Maddie/Slate upgrade

### PR DESCRIPTION
**Pull request by maddiechase**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Accelo API Documentation
+git# Accelo API Documentation
```
Comment (matthewhughes112) [inline README.md]: [B] preceding `git` here

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<!--
```
Comment (matthewhughes112) [inline .github/ISSUE_TEMPLATE.md]: [C] We have disabled issues in Git Hub, so this won't be used, but might be nice to clean up to remove clutter.

File: source/includes/_errors.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Errors
```
Comment (matthewhughes112) [inline source/includes/_errors.md]: [C] similarly to `.github/ISSUE_TEMPLATE.md`, this won't be visible as it's not included in `source/index.html.md` but there's also not really any need for the additional clutter.

File: .editorconfig [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# EditorConfig is awesome: http://EditorConfig.org
```
Comment (matthewhughes112) [inline .editorconfig]: [Q] Thoughts on using this @KurtWagner ? Since we're mostly just touching markdown I don't think it would be too much use to us.

Comment (KurtWagner) [reply .editorconfig]: If we're not using it then lets remove to avoid confusion 👍

File: README.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# Accelo API Documentation
+Accelo API Documentation
```
Comment (matthewhughes112) [inline README.md]: [B] Missing `#` (this line should be unchanged)

**Pull request merged**